pages/sending_data.rst

```diff
@@ -249,8 +249,6 @@ You can `learn more about JSONPath here <http://goessner.net/articles/JsonPath/>
 Reading from files
 ==================
 
-Graylog is currently not providing an out-of-the-box way to read log messages from files. We do however recommend two
-fantastic tools to do that job for you. Both come with native Graylog (GELF) outputs:
+Graylog itself is not able to read log messages from file, but it comes with the :ref:`Collector Sidecar <graylog-collector-sidecar>` which can used as a supervisor for tools that are build for this purpose.
```

Comment:
Maybe rephrase that altogether because with a plugin providing a file input, Graylog would be able to read files (from the local system). It's just not possible out-of-the-box.

Also missing word: "which can *be* used as"